src/web-components/comments/components/annotation-pin.ts

```diff
@@ -221,25 +223,21 @@ export class CommentsAnnotationPin extends WebComponentsBaseElement {
     };
     classes[this.horizontalSide] = true;
 
+    let style = '';
+    if (this.keepPositionRatio) {
+      style = `top: ${this.position.y * 100}%; left: ${this.position.x * 100}%;`;
+    } else {
+      style = `top: ${this.position.y * 100}px; left: ${this.position.x * 100}px;`;
+    }
+
```

Comment:
You must do these calculations from HTML Pin, because this way they will interfere with all the pin adapters we have, such as Matterport and Canvas.